sentry-core/src/test/java/io/sentry/core/SentryClientTest.kt

```diff
@@ -599,4 +636,10 @@ class SentryClientTest {
     }
 
     internal class CustomCachedApplyScopeDataHint : Cached, ApplyScopeData
+
+    internal class DiskFlushNotificationHint : DiskFlushNotification {
+        override fun markFlushed() {
+            TODO("Not yet implemented")
```

Comment:
L: Is there something to be implemented here or it'll just be empty?

Reply:
its an internal and test class `SentryClientTest.kt`

Reply:
but yeah could remove this comment